src/lxc/start.c

```diff
@@ -1278,8 +1292,6 @@ static int do_start(void *data)
 
 	/* Setup the container, ip, names, utsname, ... */
 	ret = lxc_setup(handler);
-	close(handler->data_sock[1]);
-	close(handler->data_sock[0]);
```

Comment:
These probably still need to be closed on failure paths?

Reply:
Yeah, also left-over when hacking this in for testing.

Reply:
done